### PR TITLE
Fix CubePlaytestPage.js

### PR DESCRIPTION
**Pull request by ruler501**
Removes a warning from the console of cards being null and fixes the error with useAlerts preventing it from rendering.

### CODE REVIEW

File: src/pages/CubePlaytestPage.js [hunk 1]
```diff
@@ -345,7 +345,7 @@ const CubePlaytestPage = ({ cube, cubeID, canEdit, decks, draftFormats }) => {
 
 CubePlaytestPage.propTypes = {
   cube: PropTypes.shape({
-    cards: PropTypes.arrayOf(PropTypes.object).isRequired,
+    cards: PropTypes.arrayOf(PropTypes.object),
```
Comment (phulin) [inline src/pages/CubePlaytestPage.js]: Why this change?

Comment (ruler501) [reply src/pages/CubePlaytestPage.js]: I was getting warnings in the browser that we were passing a cube without cards and I checked and we aren't using cards in this file so it didn't seem to need to be required.
